### ISSUE TITLE
Cannot restore panes started with a command

### ISSUE THREAD

**Issue by jandamm** [tmux-plugins/tmux-resurrect#331]
I'm using thoughtbot/gitsh to quickly open a tmux window for git.
But every time I need to restore my tmux session these windows are just an empty shell.
I've added `gitsh` to the `resurrect-processes. Even using all did not work.

It however does work if I create a new pane and type `gitsh` manually.

I've also tested with nvim (to verify its isn't `gitsh` specific). Even nvim cannot be restored.

Here is an example of the key binding I use for `gitsh` (using nvim instead):

```conf
bind-key g split-window -fh -p 50 -c  
        "#{pane_current_path}" nvim
```

**Comment by jandamm** [comment]
I got it to work with a bit of hacking, but it'd still be nice if it is supported by default.

The bind-key is:
```conf
bind-key g split-window -fh -p 50 -c "#{pane_current_path}" "gitsh; echo"
```
The echo is actually never executed/seen. The multiple commands just get resurrect to execute `gitsh` in the new shell.

This solves the issue of resurrect not restoring `gitsh` at all.
But after a restore the pane cannot be closed with a simple exit/`C-d`.

To fix this second issue I changed the following setting:
```conf
set -g @resurrect-processes '"gitsh->gitsh; exit" ...'
```

Now I restore `gitsh` panes correctly into `gitsh` and they still close by exiting `gitsh`.
